p4c_bm/shell.py

```diff
@@ -120,7 +124,8 @@ def main():
                                            args.public_inc_path,
                                            dump_yaml = args.dump_yaml)
     smart.render_all_files(render_dict, gen_dir,
-                           with_thrift = args.thrift)
+                           with_thrift = args.thrift,
+                           with_plugins = args.plugins)
```

Comment:
I was using with_plugins for boolean values, please rename "plugins" or "plugins_list"